src/SmartComponents/BaselinesPage/EditBaseline/helpers.js

```diff
@@ -0,0 +1,37 @@
+import jiff from 'jiff';
```

Comment:
These functions look fairly straight forward in terms of expecting an input and returning a structured output. Given that, could this include some unit tests for this file?